pyfar/dsp/dsp.py

```diff
@@ -1988,7 +1992,7 @@ def average(signal, mode='linear', caxis=None, weights=None, keepdims=False,
                                         'power',)):
```

Comment:
```suggestion
                                        )):
```
I'm not sure we need this error. I think all these things should work the same way for complex signals. Especially since the first 3 work on frequency data. And ``'power'' is defined for complex time data anyway.